templates/scripts/sync-up-version.js

```diff
@@ -0,0 +1,42 @@
+const fs = require("fs");
+var path = require("path");
+const loadJsonFile = require("load-json-file");
+const writePkg = require("write-pkg");
+const semver = require("semver");
+
+const sdk_version = require(path.join(__dirname, "../../packages/sdk/package.json")).version;
+const sdk_name = require(path.join(__dirname, "../../packages/sdk/package.json")).name;
+
+function listFile(dir, list = []) {
+  var arr = fs.readdirSync(dir);
```

Comment:
Our previous preference is `const` > `let` > `var`

Reply:
sure